index.html

```diff
@@ -209,14 +217,14 @@ <h4>Skills</h4>
     <footer>
         <div class="footer-info">
             <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
-                <form action="contact">
-                    <input type="text" name="name" placeholder="Full name" required/>
-                    <input type="text" name="email" placeholder="Email address" required/>
-                    <textarea name="comment" id="comment" cols="30" rows="5">Enter your text here</textarea>
-                    <div class="btn-center">
-                        <button class="btn btn-style submit-form" type="button" >Get in touch</button>
-                    </div>
-                </form>
+            <form action="https://formspree.io/f/form_id" method="post">
+                <ul>
+                    <li><input type="text" name="name" placeholder="Full name" maxlength="30" required /></li>
+                    <li><input type="email" name="email" placeholder="Email address" required /></li>
+                    <li><textarea name="comment" id="comment" cols="30" rows="5" maxlength="500">Enter your text here</textarea></li>
```

Comment:
- [ ] Let's make sure we don't have any list style to match the Figma design, In my opinion, It's not necessary to keep `ul` `li` but if you wish to keep let's make sure we remove the list style

Your implementation | Figma Design
:------:|:-----:
![image](https://user-images.githubusercontent.com/87689547/154068393-d21e8383-9991-4134-b368-ca7a66e474fb.png) | ![image](https://user-images.githubusercontent.com/87689547/154068793-0d474265-223c-4e01-b6f5-2d9520f05b32.png)

Reply:
Hello @Amrhub . I've done this requirement, can you check?